Plugins/ASEP/Get-ASEP.ps1

```diff
@@ -0,0 +1,176 @@
+    following options on each host:
+
+    autorunsc64.exe -accepteula -a * -h -nobanner -v -s -t -c
+
+    The options that are inlcuded in this execution are:
+
+    -a             Specifies the ASEP type to collect (See below)
+    -h             Collects several hashes of each ASEP
+    -v             Queries VirusTotal based on hash only
+    -s             Verifies signatures of all ASEPs
+    -t             Standardizes time in UTC format
+    -c             Outputs in a CSV format
+    -nobanner      Does not inlcude Autoruns banner
+    -accepteula    Does not prompt for EULA acceptance
+
+    This plugin opts for collection of all ASEPs available including:
+
+    Boot execute, codecs, appinit DLLs, Explorer addons, sidebar gadgets, image
+    highjacks, Internet Explorer addons, known DLLs, logon startups, WMI entries,
+    Winsock protocol and network providers, Office addins, printer monitor DLLs,
+    LSA security providers, Autostart services and non-disabled drivers, scheduled
+    tasks, winlogon entries
+
+    Note: If you are interested in running a different version of this command, you 
+    can simply edit the Invoke-Command entry below and adjust as needed.
+
+    Note: If you are using custom directory paths, you will need to edit the EXE 
+    checks in the script so that it does not error and exit. Best practice, use the
+    default paths provided by Power-Response.
+
+.EXAMPLE
+    Get-ASEP.ps1 -ComputerName Test-PC
+
+    PowerResponse Execution
+    Set ComputerName Test-PC
+    Run
+
+.NOTES
+    Author: 5yn@x
+    Date Created: 12/29/2018
+    Twitter: @5ynax
+    
+    Last Modified By:
+    Last Modified Date:
+    Twitter:
+  
+#>
+
+param (
+
+    [Parameter(Mandatory=$true,Position=0)]
+    [string[]]$ComputerName
+
+    )
+
+process{
+
+    #Get the root Power-Response directory - Assumes that the plugin is located in the plugins/ASEP directory
+    $PRRoot = (Get-Item $PSScriptRoot).parent.parent.FullName
+
+    #Autorunsc executable locations
+    $Autorunsc64 = "$PRRoot\Bin\autorunsc64.exe"
+    $Autorunsc32 = "$PRRoot\Bin\autorunsc.exe"
+
+    #Verify binaries exist in Bin
+    $64bitTestPath = Get-Item -Path $Autorunsc64 -ErrorAction SilentlyContinue
+    $32bitTestPath = Get-Item -Path $Autorunsc32 -ErrorAction SilentlyContinue
+
+    if (-not $64bitTestPath) {
+
+        Write-Error "Autorunsc64.exe not detected in Bin. Place 64bit executable in Bin directory and try again."
+        Exit 
+
+    } elseif (-not $32bitTestPath) {
+
+        Write-Error "Autorunsc.exe not detected in Bin. Place 32bit executable in Bin directory and try again."
+        Exit
+
+    }
+
+    #Loop through machines in $ComputerName to obtain data for each machine (if multiple machines are specified)
+    foreach ($Computer in $ComputerName) {
+
+    #Handle $ComputerName defined as localhost and not break the Plugin
+    if ($Computer -match "Localhost") {
```

Comment:
Nice catch on the `localhost` handling! One thing to consider would be using `-eq` vs `-match` in the if check.

Difference between the expressions is `-eq` looks for direct equality where `-match` looks to see if the string is anywhere in `$Computer`
Ex. `"localhost-butnotreally" -eq "Localhost" => false` but `"localhost-butnotreally" -match "Localhost" => true`

Reply:
I was considering this. I have just had bad experiences with -eq in the past that screwed with me, so i went around it. not a good answer though.

change made.